libfreerdp/codec/nsc.c

```diff
@@ -407,7 +407,7 @@ BOOL nsc_process_message(NSC_CONTEXT* context, UINT16 bpp,
 
 	if (!freerdp_image_copy(pDstData, DstFormat, nDstStride, nXDst, nYDst,
 				width, height, context->BitmapData,
-				PIXEL_FORMAT_BGRA32, 0, 0, 0, NULL))
+				PIXEL_FORMAT_BGRX32_VF, 0, 0, 0, NULL))
```

Comment:
Fortunately, the rfx codec kept PIXEL_FORMAT_XRGB32 in everywhere before the color conversion fix.